### ISSUE TITLE
Does not find yaml files inside jar files

### ISSUE THREAD

**Issue by jordanms** [earlye/dropwizard-multi-config#2]
The DefaultConfigurationReader and MultipleConfigurationProvider.globPath expect the configuration file to exists in the file system, what is not true for all cases. It should expect files to be in the classpath as well.

**Comment by earlye** [comment]
True. If you have an urgent need (days as opposed to weeks), could you
submit a pull request to add this feature? I'm fairly booked at the moment.

-- Early
